NEWS.md

```diff
@@ -272,6 +272,8 @@ This section lists changes that do not have deprecation warnings.
 Library improvements
 --------------------
 
+  * `in` now uses `isequal` to test for containment ([#9381]).
```

Comment:
This should probably be listed under breaking changes instead.

Reply:
Good point, fixed.